### PR TITLE
PA-102 - Add Delivery Model to add draft apprenticeship

### PR DESCRIPTION
**Pull request by benagain**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/IndexRequest.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@ public class IndexRequest : IAuthorizationContextModel
         public string AccountLegalEntityHashedId { get; set; }
 
         public string StartMonthYear { get; set; }
+        public DeliveryModel? DeliveryModel { get; set; }
```
Comment (Pauljgraham) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/IndexRequest.cs:16]: I don't think DeliveryModel needs to be here

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/CreateCohortRequestMapper.cs [hunk 2]
```diff
@@ -21,6 +21,7 @@ public Task<CreateCohortRequest> Map(ApprenticeViewModel source)
                 Email = source.Email,
                 DateOfBirth = source.DateOfBirth.Date,
                 Uln = source.Uln,
+                DeliveryModel = source.DeliveryModel,
```
Comment (Pauljgraham) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/CreateCohortRequestMapper.cs:24]: I think we are missing a test from CreateCohortRequestMapperTests

**Review by Pauljgraham — COMMENTED** [review]
Works great one additional line (not required) and one test needed. But it works !!!

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/IndexRequest.cs (hunk 1 above)
Comment (benagain) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/IndexRequest.cs:16]: OK, removed

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/CreateCohortRequestMapper.cs (hunk 2 above)
Comment (benagain) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/CreateCohortRequestMapper.cs:24]: This one? 

https://github.com/SkillsFundingAgency/das-employercommitments-v2/blob/29cd88a7ffe75dd4f30dccaa6a1b118aab61141e/src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Cohort/CreateCohortRequestMapperTests.cs#L75-L79

**Review by Pauljgraham — APPROVED** [review]

**Pull request merged**